### ISSUE TITLE
你好：请问你的问题解决了吗

### ISSUE THREAD

**Issue by yunyao01** [YunYang1994/tensorflow-yolov3#207]
你好：
BatchNorm / moving_variance 
['yolov3 / yolo-v3 / Conv_6 / weights'，TensorShape（[Dimension（1），Dimension（1），Dimension（1024），Dimension（255）]）] 
['conv_lbbox / weight'，TensorShape（ [Dimension（1），Dimension（1），Dimension（1024），Dimension（75）]）] 
Traceback（最近一次调用最后一次）：
文件“convert_weight.py”，第66行，in 
raise RuntimeError 
RuntimeError 

这个“长度不等的问题”，你有遇到过吗？怎么解决的？谢谢

_Originally posted by @change113 in https://github.com/YunYang1994/tensorflow-yolov3/issues/161#issuecomment-495492397_

**Comment by DeeeeepL** [comment]
类别数没设置正确吧
